.deepsource.toml

```diff
@@ -6,6 +6,7 @@ enabled = true
 
   [analyzers.meta]
   runtime_version = "3.x.x"
+  max_line_length = 120
```

Comment:
Is this parameter related to output files?

Reply:
By default python analyzer assumes limit of max 80 characters per limit. This is too low for modern applications, hence I increase it to 120